### ISSUE TITLE
Live-iso e2e test broken due to validation webhook

### ISSUE THREAD

**Issue by lentzi90** [metal3-io/cluster-api-provider-metal3#689]
**What steps did you take and what happened:**

Run the parallel e2e test: `/test-ubuntu-e2e-parallel-main`
The live-iso test fails with this:

```
• Failure [0.090 seconds]
When testing live iso [live-iso]
/home/****/tested_repo/test/e2e/live_iso_test.go:16
  Should update the BMH with live ISO [It]
  /home/****/tested_repo/test/e2e/live_iso_test.go:30

  Unexpected error:
      <*errors.StatusError | 0xc000c0c140>: {
          ErrStatus: {
              TypeMeta: {Kind: "", APIVersion: ""},
              ListMeta: {
                  SelfLink: "",
                  ResourceVersion: "",
                  Continue: "",
                  RemainingItemCount: nil,
              },
              Status: "Failure",
              Message: "admission webhook \"baremetalhost.metal3.io\" denied the request: BMC driver ipmi does not support live-iso image",
              Reason: "BMC driver ipmi does not support live-iso image",
              Details: nil,
              Code: 403,
          },
      }
      admission webhook "baremetalhost.metal3.io" denied the request: BMC driver ipmi does not support live-iso image
  occurred
```

**What did you expect to happen:**

No error should happen.

**Anything else you would like to add:**

This is most likely due to https://github.com/metal3-io/baremetal-operator/pull/1107
But it is unclear if the issue is in the test itself, in the validation webhook or somewhere else.

Initially I thought that filtering for BMHs that are controlled using redfish (instead of ipmi) would solve the issue. But this is not the case.
See #688

/kind bug

**Comment by dtantsur** [comment]
> BMC driver ipmi does not support live-iso image

Is a valid error: IPMI cannot be used with live-iso. We need to use redfish-virtualmedia or another virtual media based driver.

**Comment by lentzi90** [comment]
Fixed by #690 
/close
